src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -691,6 +691,37 @@ public void canOverrideRenewalWhenItemIsNotLoanableAndNewDueDateIsSpecified() th
       isEquivalentTo(newDueDate));
   }
 
+
+  @Test
+  public void cannotOverrideRenewalWhenDueDateIsEarlierOrSameAsCurrentLoanDueDate() throws
+    InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    DateTime loanDueDate = DateTime.now().plusWeeks(8);
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, jessica, loanDueDate);
```

Comment:
The third parameter to check out by barcode specifies when the check out occurs, not the intended due date. 

This might affect the validity of this example.

Reply:
changed to current date